test/e2e/mock-response-data/snaps/snap-binary-mocks.ts

```diff
@@ -0,0 +1,328 @@
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockHomePageSnap,
+  });
+}
+
+export async function mockImagesSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockImagesSnap,
+  });
+}
+
+export async function mockInsightsSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockInsightsSnap,
+  });
+}
+
+export async function mockInteractiveUiSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockInteractiveUiSnap,
+  });
+}
+
+export async function mockJsonRpcSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockJsonRpcSnap,
+  });
+}
+
+export async function mockJsxSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockJsxSnap,
+  });
+}
+
+export async function mockLifecycleHooksSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockLifecycleHooksSnap,
+  });
+}
+
+export async function mockLocalizationSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockLocalizationSnap,
+  });
+}
+
+export async function mockLookupSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockLookupSnap,
+  });
+}
+
+export async function mockManageStateSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockManageStateSnap,
+  });
+}
+
+export async function mockNetworkSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockNetworkSnap,
+  });
+}
+
+export async function mockNotificationSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockNotificationSnap,
+  });
+}
+
+export async function mockPreferencesSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockPreferencesSnap,
+  });
+}
+
+export async function mockSignatureInsightsSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockSignatureInsightsSnap,
+  });
+}
+
+export async function mockSimpleKeyringSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockSimpleKeyringSnap,
+  });
+}
+
+export async function mockWasmSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockWasmSnap,
+  });
+}
+
+export async function mockWebpackPluginOldSnap(mockServer: Mockttp) {
+  return createSnapMock({
+    mockServer,
+    snapNamePrefix: snapConfigurations.mockWebpackPluginSnap,
+    specificVersion: '2.0.0',
```

Comment:
in this case, we want to use this hardcoded version as it's an old snap, so then we can test the upgrade flow of this snap with the latest snap version